Transformers/README.md

```diff
@@ -1,15 +1,15 @@
-VideoTransformers
+Transformers
 ==============
 
-VideoTransformers is a simple application meant to get a new developer started using the
-OpenTok Windows SDK. For a full description, see the [Basic tutorial at the OpenTok developer
+Transformers is a simple application meant to get a new developer started using the
+OpenTok Windows SDK Transformers API. For a full description, see the [Basic tutorial at the OpenTok developer
 center](https://tokbox.com/developer/tutorials/windows/basic-video-chat/).
 
 *Important:* To use this application, follow the instructions in the
 [Quick Start](../README.md#quick-start) section of the main README file for this repository.
 
 <p class="topic-summary">
-You can use pre-built transformers in the Vonage Media Processor library or create your own custom video transformer to apply to published video.
+You can use pre-built transformers in the Vonage Media Processor library or create your own custom video transformer to apply to published video/audio.
 </p>
 
 You can use the
```

Comment:
a bit below we have a section `## Applying a video transformer from the Vonage Media Library`. It would be perfect if we added the `## Applying an audio transformer from the Vonage Media Library` from here https://github.com/vonage-technology/developer-center/blob/develop/src/documents/developer/guides/vonage-media-processor/windows/index.html.md#:~:text=%3D%20transformers%3B-,Applying%20an%20audio%20transformer%20from%20the%20Vonage%20Media%20Library,-Note%3A%20This